lib/index.js

```diff
@@ -131,11 +131,15 @@ var handleFailure = (options, reactEl, type, props, failureMsg) => {
   var includeSrcNode = options && (options.includeSrcNode || false);
   var warningPrefix = (options && options.warningPrefix) || '';
   var reactComponent = reactEl._owner;
+  
+  // Validate component name against React Version
+  // Version <= 15 use reactComponent.getName()
+  // Version >= 16 use reactComponent.type.name
+  var reactComponentName = Boolean(reactComponent.type) ? reactComponent.type.name : reactComponent.getName();
```

Comment:
The calls to `reactComponent` here assume that it is defined, which it may not be. This could potentially be what is breaking the build.

Reply:
Unfortunately the issues are a lot deeper than that. Using `.getName()` instead of `.type.name` isn't all that will be required to get this plugin working for React 16.